opm/simulators/flow/MissingFeatures.cpp

```diff
@@ -148,6 +148,41 @@ namespace MissingFeatures {
             "CPR",
             "CRITPERM",
             "DATE",
+            "DATES",
```

Comment:
Ehhh - this was a bit aggressive; many of these keywords are supported. Please update to only include the list of keywords merged in: https://github.com/OPM/opm-common/pull/966